src/integration_tests/regression_test.py

```diff
@@ -3,6 +3,7 @@
 from __future__ import annotations
 
 import math
+import numpy as np
```

Comment:
This import requires additional dev dependency in pyproject.toml.

Reply:
Hi Odashi, I added numpy version in pyproject.toml, let me know if the version is suitable for the project.